shared/models.rst

```diff
@@ -0,0 +1,32 @@
+.. 
+.. xxxxx
+..
+
+
+.. TODO: Make some important callouts for modeing reusable across collections.
+
+
+
+.. models-churn-propensitity-tiers-repeat-start
+
+For repeat customers, groupings include the following tiers:
+
+* Active (likelihood to purchase is greater than 60%)
+* Cooling down (likelihood to purchase is between than 50% and 60%)
+* At risk (likelihood to purchase is between than 35% and 50%)
+* Highly at risk (likelihood to purchase is between than 20% and 35%)
+* Lost (likelihood to purchase is less than 20%)
+
+.. models-churn-propensitity-tiers-repeat-end
+
+.. models-churn-propensitity-tiers-onetime-start
+
+For one-time buyers, groupings include the following tiers:
```

Comment:
One thing to note -- this window of 60 days is actually configurable via the `:amperity.prediction.model.pclv.ensemble/lifecycle-status-lapsed-recency-window` key. If you set it to, say, 30, then the window below would be 

Active (purchased within 30 days)
Cooling down (purchased between 30-60 days) 
etc.

You can set this config option in the advanced settings of a predictive model